Plugins/Public/playercntl_plugin/Main.cpp

```diff
@@ -903,7 +903,7 @@ USERCMD UserCmds[] =
 	{ L"/pos",			MiscCmds::UserCmd_Pos,			L"Usage: /pos" },
 	{ L"/stuck",		MiscCmds::UserCmd_Stuck,		L"Usage: /stuck" },
 	{ L"/droprep",		MiscCmds::UserCmd_DropRep,		L"Usage: /droprep" },
-	{ L"/dice",			MiscCmds::UserCmd_Dice,			L"Usage: /dice [max]" },
```

Comment:
Do we really want to remove existing commands?

Reply:
I mean. It could stay. But why, when the new one does the very same thing and more?

Reply:
Can we not alias /dice to the new command?

Reply:
A fair point. 
/dice has been re-added, and pointed to the same method.

/roll is being kept in, as that's the command generally used everywhere else on the internet for this sort of operation.